web/package.json

```diff
@@ -6,7 +6,13 @@
     "axios": "^0.18.0",
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
-    "react-scripts": "1.1.4"
+    "react-redux": "^5.0.7",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
+    "react-scripts": "1.1.4",
+    "redux": "^4.0.0",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
```

Comment:
alphabetize your imports you god damn neanderthal

Reply:
It is?!?

Reply:
Christ... Im an idiot. Nevermind :/ hah